tests/beta/queuing/test_beta_measure.py

```diff
@@ -0,0 +1,121 @@
+)
+class TestBetaStatistics:
+    """Tests for annotating the return types of the statistics functions"""
+
+    @pytest.mark.parametrize(
+        "op", [qml.PauliX, qml.PauliY, qml.PauliZ, qml.Hadamard, qml.Identity],
+    )
+    def test_annotating_obs_return_type(self, stat_func, return_type, op):
+        """Test that the return_type related info is updated for an expval call"""
+        with qml._queuing.AnnotatedQueue() as q:
+            A = op(0)
+            stat_func(A)
+
+        assert q.queue == [A]
+        assert q.get_info(A) == {"return_type": return_type}
+
+    def test_annotating_tensor_hermitian(self, stat_func, return_type):
+        """Test that the return_type related info is updated for an expval call
+        when called for an Hermitian observable"""
+
+        mx = np.array([[1, 0], [0, 1]])
+
+        with qml._queuing.AnnotatedQueue() as q:
+            Herm = qml.Hermitian(mx, wires=[1])
+            stat_func(Herm)
+
+        assert q.queue == [Herm]
+        assert q.get_info(Herm) == {"return_type": return_type}
+
+    @pytest.mark.parametrize(
+        "op1,op2",
+        [
+            (qml.PauliY, qml.PauliX),
+            (qml.Hadamard, qml.Hadamard),
+            (qml.PauliY, qml.Identity),
+            (qml.Identity, qml.Identity),
+        ],
+    )
+    def test_annotating_tensor_return_type(self, op1, op2, stat_func, return_type):
+        """Test that the return_type related info is updated for an expval call
+        when called for an Tensor observable"""
+        with qml._queuing.AnnotatedQueue() as q:
+            A = op1(0)
+            B = op2(1)
+            tensor_op = BetaTensor(A, B)
+            stat_func(tensor_op)
+
+        assert q.queue == [A, B, tensor_op]
+        assert q.get_info(A) == {"owner": tensor_op}
+        assert q.get_info(B) == {"owner": tensor_op}
+        assert q.get_info(tensor_op) == {"owns": (A, B), "return_type": return_type}
+
+@pytest.mark.parametrize(
+    "stat_func", [expval, var, sample]
+)
+class TestBetaStatisticsError:
+    """Tests for errors arising for the beta statistics functions"""
+
+    def test_not_an_observable(self, stat_func):
+        """Test that a QuantumFunctionError is raised if the provided
+        argument is not an observable"""
+        dev = qml.device("default.qubit", wires=2)
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.RX(0.52, wires=0)
+            return stat_func(qml.CNOT(wires=[0, 1]))
+
+        with pytest.raises(QuantumFunctionError, match="CNOT is not an observable"):
+            res = circuit()
+
+
+class TestBetaProbs:
+    """Tests for annotating the return types of the probs function"""
+
+    @pytest.mark.parametrize("wires", [[0], [0, 1], [1, 0, 2]])
+    def test_annotating_probs(self, wires):
+
+        with qml._queuing.AnnotatedQueue() as q:
+            probs(wires)
+
+        assert len(q.queue) == 1
+        assert isinstance(q.queue[0], qml.Identity)
```

Comment:
Hmm, this feels like the best time to break with an old, somewhat ad-hoc behaviour (also seen in the states return PR).

Since `probs` doesn't really require any observable, we shouldn't have to queue `Identity` (it is essentially a dummy op). I think I'd much rather update the codebase to remove this (historical) restriction than to try to force its continued use